application/common/application_data.h

```diff
@@ -158,6 +159,7 @@ class ApplicationData : public base::RefCountedThreadSafe<ApplicationData> {
   bool LoadVersion(base::string16* error);
   bool LoadDescription(base::string16* error);
   bool LoadWindowSetting(base::string16* error);
+  bool LoadDisplayMode(base::string16* error);
```

Comment:
I think the display mode is the same as window settings?
I think it's a application context setting, if a new top-level browsing context is opened, the display mode should still be applied.